VERSION

```diff
@@ -1 +1 @@
-v3.4.13-bootstrap-4
+v3.4.18
```

Comment:
can we also change the version with something appropriate as `v3.4.18` might be ambiguous as this is the custom image just like we used to add  `bootstrap` suffix

Reply:
Is this make sense ? @aaronfern @abdasgupta

Reply:
I think it's fine without the suffix as we're mirroring the etcd version in our custom image version
We could further add a suffix whenever we make changes to the bootstrap file

However, no strong opinion from me